.pre-commit-config.yaml

```diff
@@ -31,6 +31,13 @@ repos:
   - id: debug-statements
   - id: requirements-txt-fixer
 
+- repo: https://github.com/ansys/pre-commit-hooks
+  rev: v0.4.3
+  hooks:
+    - id: add-license-headers
+      args:
+      - --start_year=2020
```

Comment:
Adding this as the start year after taking a look to the history of the `LICENSE` file, see https://github.com/ansys/pydpf-core/commits/master/LICENSE